NewForce-at-Mountwest/bangazon-site-the-blue-ferrets
# Sarah-Order  - Complete Order

# Description

Can complete order. View message for empty cart

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions
Log in as admin. 
Add a product to your order.
Click view cart.
Click checkout at the bottom.
Select payment type
Click complete order.
-Should go to list of completed orders.
Click on view cart
Should see a message that says your cart is empty


# Checklist:
